src/global-models/index.ts

```diff
@@ -0,0 +1,60 @@
+export interface Episode {
```

Comment:
I'm not sure if these interfaces are related to global-models. Looks like they are mostly related to `SWAPI`. Think about it. Probably it makes sense to refactor file structure a little bit